### ISSUE TITLE
TypeError: Object.values requires that input parameter not be null or undefined

### ISSUE THREAD

**Issue by mrousavy** [software-mansion/react-native-reanimated#1245]
## Description

I'm getting `TypeError: Object.values requires that input parameter not be null or undefined` when using a `useAnimatedStyle` hook

### Screenshots

![Screenshot 2020-09-17 at 23 41 40](https://user-images.githubusercontent.com/15199031/93531870-34f64780-f940-11ea-8f60-6e9c60056b25.png)

## Steps To Reproduce

1. Create `useAnimatedStyle` hook
2. Receive error

### Expected behavior

I expect the animated style to work

### Actual behavior

It throws an error

## Snack or minimal code example

> (working on a repro)

## Package versions

- React: 16.13.1
- React Native: 0.63.2
- React Native Reanimated: 2.0.0-alpha.6

**Comment by terrysahaidak** [comment]
Hi. Make sure you used babel plugin as well as run packager with `--reset-cache`

**Comment by mrousavy** [comment]
My bad! I've just double checked my babel config and noticed I only added reanimated plugin in production environment:

My babel config:
```js
module.exports = {
	presets: ['module:metro-react-native-babel-preset'],
	env: {
		production: {
			plugins: ['transform-remove-console', 'react-native-reanimated/plugin'],
		},
	},
};
```

Fixed it by adding it in the default plugins list. Thanks!

**Comment by alex-adestech** [comment]
My App was rejected on iOS because of this issue. It's only happening in release mode. Should I add another config besides the "plugins" object? Or maybe downgrade to alpha 5

**Comment by terrysahaidak** [comment]
Can you show your babel config?

Also, try to run the app in using release schema.

**Comment by alex-adestech** [comment]
Sure.

This is my babel config:
```
module.exports = {
  presets: ['module:metro-react-native-babel-preset'],
  plugins: [
    'transform-remove-console',
    'react-native-reanimated/plugin',
  ],
  env: {
		production: {
			plugins: ['transform-remove-console', 'react-native-reanimated/plugin'],
		},
	},
};
```

This is the error shown in xcode running in release schema:
```
[javascript] TypeError: Object.values requires that input parameter not be null or undefined

This error is located at:
    in Unknown
    in RCTView
    in View
    in RCTView
    in View
    in RCTModalHostView
    in y
    in RCTView
    in View
    in P
    in A
    in RCTView
    in View
    in RCTView
    in View
    in RCTView
    in View
    in I
    in A
    in RCTSafeAreaView
    in ForwardRef
    in RCTView
    in View
    in n
    in n
    in b
    in c
    in Unknown
    in RCTView
    in View
    in RCTView
    in View
    in C
[javascript] TypeError: Object.values requires that input parameter not be null or undefined
```

This is the code that crashes
```
const offset = useSharedValue(height);
  const backgroundColor = useSharedValue(0);
  const scale = useSharedValue(minScale);
  const opacity = useSharedValue(0);

  const defaultSpringStyles = useAnimatedStyle(() => {
    return {
      backgroundColor: backgroundColor.value == 0 ? colors.green : colors.background,
      transform: [
        {
          translateY: offset.value,
        },
        {
          scaleX: scale.value,
        },
      ],
    };
  });

  const opacityAnimation = useAnimatedStyle(() => {
    return {
      opacity: opacity.value,
    };
  });

  setTimeout(() => {
    offset.value = withTiming(0, { duration: 300 });
    setTimeout(() => {
      scale.value = withTiming(1, { duration: 700 });
      setTimeout(() => {
        backgroundColor.value = withTiming(1, { duration: 50 });
      }, 30);
      setTimeout(() => {
        opacity.value = withTiming(1, { duration: 250 });
      }, 700);
    }, 500);
  }, 500);
```

**Comment by terrysahaidak** [comment]
I guess your packager has some sort of cache. Could you try to clear all the cache and build again?

**Comment by alex-adestech** [comment]
There's also this log in the console
```
*** First throw call stack:
(
	0   CoreFoundation                      0x00007fff2043a126 __exceptionPreprocess + 242
	1   libobjc.A.dylib                     0x00007fff20177f78 objc_exception_throw + 48
	2   Parlaytor 2                         0x000000010cadf3be RCTFormatError + 0
	3   Parlaytor 2                         0x000000010cb6110a -[RCTExceptionsManager reportFatal:stack:exceptionId:suppressRedBox:] + 503
	4   Parlaytor 2                         0x000000010cb61b88 -[RCTExceptionsManager reportException:] + 1658
	5   CoreFoundation                      0x00007fff2044092c __invoking___ + 140
	6   CoreFoundation                      0x00007fff2043dde6 -[NSInvocation invoke] + 303
	7   CoreFoundation                      0x00007fff2043e077 -[NSInvocation invokeWithTarget:] + 70
	8   Parlaytor 2                         0x000000010cbc6e7d ___ZN8facebook5react15ObjCTurboModule23performMethodInvocationERNS_3jsi7RuntimeENS0_26TurboModuleMethodValueKindEPKcP12NSInvocationP14NSMutableArrayi_block_invoke + 128
	9   libdispatch.dylib                   0x000000010e80a8ac _dispatch_call_block_and_release + 12
	10  libdispatch.dylib                   0x000000010e80ba88 _dispatch_client_callout + 8
	11  libdispatch.dylib                   0x000000010e812356 _dispatch_lane_serial_drain + 796
	12  libdispatch.dylib                   0x000000010e813027 _dispatch_lane_invoke + 439
	13  libdispatch.dylib                   0x000000010e81ee90 _dispatch_workloop_worker_thread + 882
	14  libsystem_pthread.dylib             0x00007fff5dcd8a3d _pthread_wqthread + 290
	15  libsystem_pthread.dylib             0x00007fff5dcd7b77 start_wqthread + 15
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Comment by alex-adestech** [comment]
Sure. Let me try again.

**Comment by alex-adestech** [comment]
@terrysahaidak I already cleaned npm, watchman, xcode but still the same error :( any ideas?

**Comment by terrysahaidak** [comment]
Could you try removing production env from babel config then? To make sure it doesn't break it

**Comment by alex-adestech** [comment]
Solved it!

I didn't realized I had 2 babel files, so at the end this worked in release mode:

.babelrc file
```
{
  "presets": ["module:metro-react-native-babel-preset"],
  "plugins": [
    "transform-remove-console",
    "react-native-reanimated/plugin",
  ],
}
```

babel.config.js file
```
module.exports = {
  presets: ['module:metro-react-native-babel-preset'],
  plugins: [
    'transform-remove-console',
    'react-native-reanimated/plugin',
  ],
};
```
